index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>LAB-2</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/topnav.css" />
+    <link rel="stylesheet" href="css/sidenav.css" />
+    <link rel="stylesheet" href="css/nav-btn.css" />
+    <link rel="stylesheet" href="css/menu.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"/>
+  </head>
+  <body>
+    <nav class="menu-btn" id="main-menu" onclick="clickMenu()">
+      <i class="fas fa-bars"></i>
+    </nav>
+    <nav class="sidenav" id="side-nav">
+      <div class="nav-btns">
```

Comment:
1) ul > li > a
2) https://stackoverflow.com/questions/11078509/how-to-increase-the-clickable-area-of-a-a-tag-button